sse2neon.h

```diff
@@ -1139,28 +1163,45 @@ FORCE_INLINE __m128i _mm_shuffle_epi32_default(__m128i a,
         }                                                \
         ret;                                             \
     })
+#endif /* generic */
 
-// Shuffles the upper 4 signed or unsigned 16 - bit integers in a as specified
+// Shuffles the lower 4 signed or unsigned 16 - bit integers in a as specified
 // by imm.  https://msdn.microsoft.com/en-us/library/13ywktbs(v=vs.100).aspx
```

Comment:
Could you help us by updating the correct link again? Thanks~
https://docs.microsoft.com/en-us/previous-versions/visualstudio/visual-studio-2010/y41dkk37(v=vs.100)